bin/bout-boutdata-restart-create

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# PYTHON_ARGCOMPLETE_OK
+
+import argparse
+try:
+    import argcomplete
+except ImportError:
+    argcomplete=None
+
+parser = argparse.ArgumentParser("Create restart files from data (dmp) files.")
+parser.add_argument("path", type=str, default=...
+    ,help="Path to original restart files (default: \"data\")")
+parser.add_argument("--averagelast", type=int, default=...
+    ,help="Number of time points (counting from `final`, inclusive) to"
+     " average over (default is 1 i.e. just take last time-point)")
+parser.add_argument("--final", type=int, default=...
+    ,help="The last time point to use (default is last, -1)")
+parser.add_argument("--output", type=str, default=...
+    ,help="Path to write new restart files (default: current directory)")
+parser.add_argument("--informat", type=str, default=...
+    ,help="File extension of original files (default: \"nc\")")
+parser.add_argument("--outformat", type=str, default=...
+    ,help="File extension of new files (default: use the same as `informat`)")
+
+if argcomplete:
+    argcomplete.autocomplete(parser)
+
+# late import for faster auto-complete
+from boutdata.restart import create
+
+args = parser.parse_args()
+
+# remove non-set optional arguments
+dic=args.__dict__
+
+if dic["output"] == ... and dic["path"] != ...:
+    dic["output"] = dic["path"]
+
+for k in dic:
+    if dic[k] == ...:
+        del dic[k]
```

Comment:
Set sensible defaults in the `parser.add_argument` and this isn't necessary. Similarly for the `dic = args.__dict__` above -- that just looks suspicious

Reply:
I removed this part.
That changes the default output dir (which was previously the same as input), but as this is anyway the default of the function, I think this is fine.